### PR TITLE
prevent opening the same file twice, but also help the user by focusi…

### PR DESCRIPTION
**Pull request by matt-land**
prevent opening the same file twice, but also help the user by focusing the tab they attempted to open

### CODE REVIEW

File: mu/logic.py [hunk 1]
```diff
@@ -566,6 +566,15 @@ def load(self):
         """
         path = self._view.get_load_path(get_workspace_dir())
         logger.info('Loading script from: {}'.format(path))
+        # see if file is open first
+        for widget in self._view.widgets:
+            if widget.path is None:  # this widget is an unsaved buffer
+                continue
+            if path in widget.path:
+                self._view.show_message('Requested file is already open.')
```
Comment (ZanderBrown) [inline mu/logic.py]: How about 'that file is already open' or include the filename.
Remember that the message needs to be clear perferably using simple language but without appearing condescending

Comment (matt-land) [inline mu/logic.py]: I changed the wording to "That file is already open"
The other suggestion-I was not sure how to express the filename to the user while being simple.

Comment (ZanderBrown) [inline mu/logic.py]: Something along the lines of 'The file "myprog.py" is already open' (without file path)
But that's getting petty over a single sentence in a solution to a problem we probably should have anticipated

File: mu/interface.py [hunk 2]
```diff
@@ -653,7 +653,7 @@ def connect_zoom(self, widget):
     @property
     def current_tab(self):
         """
-        Returns the currently focused tab.
+        Returns the currently focussed tab.
```
Comment (ZanderBrown) [inline mu/interface.py]: Is it supposd to have a double s?

File: mu/logic.py (hunk 1 above)
Comment (matt-land) [inline mu/logic.py]: OK

File: mu/interface.py (hunk 2 above)
Comment (matt-land) [inline mu/interface.py]: My IDE flagged it, so I fixed it. When I googled, it seemed valid.  Since I had doubts I pulled it.

**Comment by ntoll** [comment]
Thanks for the contributions and many apologies for taking so long to getting around to reviewing/merging the PR. I had to make a few minor changes to fix PEP8 and test coverage problems.

**Pull request merged**